### ISSUE TITLE
Updating Cesium and Resium causes issues all over app

### ISSUE THREAD

**Issue by AdamFarhan** [darwin-education/resium#133]
Hello, I recently updated Cesium/Resium to make use of the new PartialEllipsoid shape. After updating to cesium version 1.63.1, and resium version 1.9.0, my app is no longer functional.

Without changing any code, I get the following issue: 
`TypeError: cesium__WEBPACK_IMPORTED_MODULE_2___default.a is undefined`

I think this is related to how I import cesium (`import Cesium from 'cesium'`).

When I avoid using Cesium directly like that, I get issues with refs:
`<Viewer ref={ref => { this.viewerComponent = ref; }}>` this is taken straight from the Resium ref example, however it now returns null.

Is there any kind of information I can find in regards to updating my project, or am I forced to refactor everything?

**Comment by rot1024** [comment]
Maybe unnecessary to refactor code which uses Resium (excludes Cesium's import statements).

1. Cesium often releases versions including breaking changes. In Cesium v1.53, JS bundler has been changed and the default is not exported, so it needs refactoring Cesium's import statements.

2. I recommend to check `ref` is not null when using ref property with a function, because `ref` may be null, for example when the component is unmounted. This is due to React specification.

```jsx
<Viewer ref={ref => { this.viewerComponent = ref; }} />
```

3. In Resium v1.9.0, I found a bug that cesiumElement won't be exposed when component is mounted. I fixed it and Resium v1.9.1 has been released. Try it!
